src/plone/app/multilingual/profiles/default/types/LIF.xml

```diff
@@ -21,7 +21,7 @@
   </property>
   <property name="default_view_fallback">False</property>
   <property name="add_permission">cmf.ManagePortal</property>
-  <property name="klass">plone.dexterity.content.Container</property>
+  <property name="klass">plone.app.multilingual.content.lif.LanguageIndependentFolder</property>
```

Comment:
So LIF folders created before this change have class `Container` and LIF folders created after this change have class `LanguageIndependentFolder`. That seems an unwanted difference. Also, it may create upgrade problems if we ever decide to remove this class, or if plone.app.multilingual is no longer included by default. Seems not so nice for a class that is just meant for "nicer output".

Reply:
Agree, this is a different, immersive change. Also, I do not see where this class exists. 
Looking at https://github.com/plone/plone.app.multilingual/blob/master/src/plone/app/multilingual/content/lif.py there is no such class.

Reply:
true.

i was a little over the top and removed this part - i might create a separate issue for that.

reason was:
when ever i create a contenttype i create a class based on container or item.
just to see
```python
print content_obj
<Container at ...>
```
vs
```python
print content_obj
<MyAwesomeContentType at ...>
```

this is imho totally cosmetical. i do that out of habit.